docs/literate/main.jl

```diff
@@ -6,7 +6,9 @@ using CairoMakie
 using Statistics
 using LuxCUDA
 using Revise
+using testdata
```

Comment:
there is no package `testdata` that I'm aware of - are you sure this is what you need?